owncloudApp/src/main/java/com/owncloud/android/presentation/manager/AvatarManager.kt

```diff
@@ -65,12 +65,10 @@ class AvatarManager : KoinComponent {
 
         val capabilities: GetStoredCapabilitiesUseCase by inject()
         val result = capabilities.execute(GetStoredCapabilitiesUseCase.Params(accountName = account.name))
-        val capabilityIsNotUnknown = result?.filesSharingUserProfilePicture?.isUnknown ?: false
         val capabilityIsNotFalse = result?.filesSharingUserProfilePicture?.isFalse ?: false
 
-        // Avatar not found in disk caché, fetch from server.
-
-        if (fetchIfNotCached && (!capabilityIsNotUnknown || !capabilityIsNotFalse)) {
+        // Avatar not found in disk cache, fetch from server.
+        if (fetchIfNotCached && !capabilityIsNotFalse) {
```

Comment:
What does `!capabilityIsNotFalse` mean? 🤔

Reply:
It means that the capability is not null and its value is false, maybe I should change the name to `capabilityIsFalse` instead of `capabilityIsNotFalse` I think it could be more understandable.